mmhuman3d/models/body_models/star.py

```diff
@@ -259,35 +258,65 @@ def forward(self,
         v_shaped = torch.matmul(shapedirs, beta).view(-1, 6890, 3) + v_template
         J = torch.einsum('bik,ji->bjk', [v_shaped, self.J_regressor])
 
-        pose_quat = self.normalize_quaternion(body_pose.view(-1, 3)).view(
-            batch_size, -1)
+        if len(body_pose.shape) == 4:
+            # the shape of body pose is rot matrix, convert to angle_axis
+            body_pose = rotation_matrix_to_angle_axis(
+                body_pose.view(-1, 3, 3)).view(batch_size, -1, 3)
+
+        pose_quat = self.quat_feat(body_pose.view(-1, 3)).view(batch_size, -1)
         pose_feat = torch.cat((pose_quat[:, 4:], beta[:, 1]), 1)
 
-        R = aa_to_rotmat(body_pose.view(-1, 3)).view(batch_size, 24, 3, 3)
+        R = self.rodrigues(body_pose.view(-1, 3)).view(batch_size, 24, 3, 3)
```

Comment:
Is there any reason why we are not using `aa_to_rotmat`?

Reply:
I've only verified the modified code by visualization. And I'm not sure if `aa_to_rotmat` and `self.rodrigues` can get the same result.  I'll confirm it later and change it back if there's no problem.

Reply:
> Is there any reason why we are not using `aa_to_rotmat`?

Sorry. After rechecking the file, there is indeed no difference between the two functions `aa_to_rotmat` and `self.rodrigues`, I have modified the code and recommitted it.